packages/stores/types.ts

```diff
@@ -176,9 +206,13 @@ type TClientStore = {
         trading_platform_mt5_password_reset: string;
     };
     email: string;
+    fetchStatesList: () => Promise<unknown>;
```

Comment:
```suggestion
import { StatesListResponse } from '@deriv/api-types';

    fetchStatesList: () => Promise<StatesListResponse>;
```

:thinking:

Reply:
I was using StatesList type but it was throwing error so i changed it to unknown.I will change it with this.Thanks

Reply:
oh got it, what about this type?
```suggestion
    fetchStatesList: <T,>() => Promise<T>;
```

Reply:
Implemented.Thanks

Reply:
Thank @yaswanth-deriv :pray: :white_flower: